runtime-modules/content/src/lib.rs

```diff
@@ -1099,8 +1215,29 @@ decl_module! {
             list: Vec<T::VideoId>
         ) {
             // can only be set by lead
+            ensure_actor_authorized_to_update_or_delete_channel_assets::<T>(
+                origin,
+                &actor,
+                // The channel owner will be..
+                &Self::actor_to_channel_owner(&actor)?,
+            )?;
 
-            Self::not_implemented()?;
+            Self::ensure_videos_exist(&list)?;
+
+            //
+            // == MUTATION SAFE ==
+            //
+
+            // Update videos featured status
```

Comment:
Lets just drop is_featured. we can add it in future if we decide we need this state in the runtime.

is_censored might be good to keep in the runtime if for example we want to prevent an owner to add more content to a channel while it is censored.

Reply:
Ok, great https://github.com/Joystream/joystream/pull/2194/commits/ffaebf857437c64b00651b775ffaac4dd6f0be3d